### PR TITLE
KM DCP Release 14 Updates

### PR DESCRIPTION
**Pull request by kayleemathews**

### CODE REVIEW

File: content/community-update/community-update/dcp-updates.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### February 28, 2022
```
Comment (ekiernan) [inline content/community-update/community-update/dcp-updates.md]:
```suggestion
#### March 31, 2022
```

File: content/community-update/community-update/dcp-updates.md [hunk 2]
```diff
@@ -6,6 +6,37 @@ title: DCP Updates
...
+
+### Release Highlights
+
+* A new managed access project, [A cellular census of human lungs identifies novel cell states in health and in asthma](https://data.humancellatlas.org/explore/projects/c0518445-3b3b-49c6-b8fc-c41daa4eacba), is available in the Browser.
+
+* A new COVID-19 dataset has been added to the DCP as part of the project [Single-cell RNA sequencing of urinary cells reveals distinct cellular diversity in COVID-19-associated AKI](https://data.humancellatlas.org/explore/projects/c6a50b2a-3dfd-4ca8-9b48-3e682f568a25).
+
+* New data from across the globe has been added to the DCP, including datasets from Africa, Asia, and Europe.
```
Comment (ekiernan) [inline content/community-update/community-update/dcp-updates.md]:
```suggestion
* New raw data derived from populations in Africa, Asia, and Europe are now available.
```

**Review by ekiernan — APPROVED** [review]
This LGTM, but please check the two suggestions, especially the date.

File: content/community-update/community-update/dcp-updates.md (hunk 1 above)
Comment (kayleemathews) [reply content/community-update/community-update/dcp-updates.md]: Should this be when the release actually went live? I'm assuming yes, which isn't what I originally thought. My bad!

Comment (ekiernan) [reply content/community-update/community-update/dcp-updates.md]: I think we've been doing the day the release notes are written, which is generally the same day it's live or close to it.

Comment (kayleemathews) [reply content/community-update/community-update/dcp-updates.md]: Got it. Thanks!

Comment (kayleemathews) [reply content/community-update/community-update/dcp-updates.md]: Updated.

**Pull request merged**